src/main/java/com/karankumar/bookproject/backend/utils/TestData.java

```diff
@@ -157,13 +157,4 @@ public static PredefinedShelf generateRandomPredefinedShelf(
         }
         return books;
     }
-
-    public static List<PredefinedShelf> createPredefinedShelves(List<Book> books) {
-        return Stream.of(PredefinedShelf.ShelfName.values())
-              .map(shelfName -> {
-                  PredefinedShelf shelf = new PredefinedShelf(shelfName);
-                  shelf.setBooks(new HashSet<>(books));
-                  return shelf;
-              }).collect(Collectors.toList());
-    }
```

Comment:
Could you please tell why did we remove this?

Reply:
It is moved to `PredefinedShelfService` so only this service is responsible for creating new predefined shelves